### PR TITLE
docs: Release @excalidraw/excalidraw@0.5.0 🎉 

### PR DESCRIPTION
**Pull request by ad1992**
- [x] Merge #3286 and #3288 
- [x] Add date 
- [x] Update CSB example & testing
- [x] Release the version

pushed [here](https://y0ld3.csb.app/)

### CODE REVIEW

File: src/packages/excalidraw/README.md [hunk 1]
```diff
@@ -356,7 +361,10 @@ export default function IndexPage() {
 }
 ```
 
-### Props
+</details>
+
+<details id="props">
+<summary><strong style="font-size: 1.25em;">Props</strong></summary>
```
Comment (ad1992) [inline src/packages/excalidraw/README.md:367]:
```suggestion
<summary><strong>Props</strong></summary>
```

File: src/packages/excalidraw/README.md [hunk 2]
```diff
@@ -539,7 +547,10 @@ This prop controls Excalidraw's theme. When supplied, the value takes precedence
 
 This prop sets the name of the drawing which will be used when exporting the drawing. When supplied, the value takes precedence over `intialData.appState.name`, the `name` will be fully controlled by host app and the users won't be able to edit from within Excalidraw.
 
-### Extra API's
+</details>
+
+<details id="extra-apis">
+<summary><strong style="font-size: 1.25em;">Extra API's</strong></summary>
```
Comment (ad1992) [inline src/packages/excalidraw/README.md:553]:
```suggestion
<summary><strong>Extra API's</strong></summary>
```

File: src/packages/excalidraw/README.md [hunk 3]
```diff
@@ -28,7 +28,8 @@ If you want to load assets from a different path you can set a variable `window.
 
 [Try here](https://codesandbox.io/s/excalidraw-ehlz3).
 
-### Usage
+<details id="usage">
+<summary><strong style="font-size: 1.25em;">Usage</strong></summary>
```
Comment (ad1992) [inline src/packages/excalidraw/README.md:32]:
```suggestion
<summary><strong>Usage</strong></summary>
```

**Comment by ad1992** [comment]
In [codesandbox](https://y0ld3.csb.app/) when installing library the site doesn't load but if i open in a new tab it works fine. This might be due to `history.pushState`, should we try setting `window.location` instead @dwelle ?  In local it works fine though.

**Comment by dwelle** [comment]
> In [codesandbox](https://y0ld3.csb.app/) when installing library the site doesn't load but if i open in a new tab it works fine. This might be due to `history.pushState`, should we try setting `window.location` instead @dwelle ? In local it works fine though.

What `history.pushState` are you referring to? It seems to be caused by opening the new tab via `target="_excalidraw"` instead of `target="_blank"`. I'm not sure what the CSB is doing that this is an issue. Setting `target="_excalidraw"` is useful because it then reuses the same tab when you install multiple libraries.

**Review by dwelle — APPROVED** [review]

**Pull request merged**